### PR TITLE
Feature/change design 2 40638891

### PR DESCRIPTION
**Pull request by sanjanadhikari1**
- [ ] Update the design (General screens - 2)

[https://freedcamp.com/view/2956183/tasks/40638891](url)

### CODE REVIEW

**Review by CrazyOptimist — APPROVED** [review]

**Review by ngvtuan — COMMENTED** [review]
ui issue 1
ui issue 2

File: src/features/dashboard/Notification.tsx [hunk 1]
```diff
@@ -1,24 +1,63 @@
...
+    >
+      <div
+        className={cn(
+          'text-base text-gray-4a',
+          read ? 'font-medium' : 'font-extrabold',
+        )}
+      >
+        {message}
+      </div>
+      {/* <DateTimeWithDot value={date} className='text-sm text-gray-a0 ml-auto' /> */}
```
[the remark this answers is not in the document]
Comment (sanjanadhikari1) [reply src/features/dashboard/Notification.tsx:53]: fixed

File: src/features/dashboard/PortfolioItem.tsx [hunk 2]
```diff
@@ -6,35 +6,28 @@ export type TPortfolioItemProps = {
   author: string
   price: number
   currencyName: string
+  type?: string
```
[the remark this answers is not in the document]
Comment (sanjanadhikari1) [reply src/features/dashboard/PortfolioItem.tsx:9]: fixed

File: src/features/dashboard/DashboardPage.tsx [hunk 3]
```diff
@@ -1,23 +1,24 @@
...
+import Radio from 'common/components/Radio'
 import NotificationModal from './dashboardModals/notificationModal'
 import Link from 'common/components/Link'
 import notificationData from './dashboardModals/notificationModal.data'
 
 const date = dayjs('2021-04-04')
 
 const walletBalance = 32000
-const currencyName = 'PSL'
+const currencyName = 'PSTL'
```
[the remark this answers is not in the document]
Comment (sanjanadhikari1) [reply src/features/dashboard/DashboardPage.tsx:21]: fixed

File: src/legacy/Routes.tsx [hunk 4]
```diff
@@ -502,6 +503,16 @@ class RouteApp extends React.Component<any, any> {
         <AboutModal />
         <SquooshToolModal />
         <GlitchImageModal />
+        {info?.version && (
```
[the remark this answers is not in the document]
Comment (sanjanadhikari1) [reply src/legacy/Routes.tsx:506]: fixed

**Review by ngvtuan — APPROVED** [review]

**Pull request merged**